modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithMediaParagraphBehavior.php

```diff
@@ -192,6 +203,35 @@ public function view(array &$build, Paragraph $paragraph, EntityViewDisplayInter
 
   }
 
+  /**
+   * Get settings from paragraph and override defaults.
+   */
+  private function getParagraphBackgroundSettings(ParagraphInterface $paragraph) {
+
+    // Get plugin configuration.
+    $config = $this->getSettings($paragraph);
+    $bg_attachment = 'scroll';
+    if (!empty($config['bg_attachment'])) {
+      $bg_attachment = 'fixed';
+    }
+    return [
+      'bg_image_selector' => '#' . $paragraph->bundle() . "-" . $paragraph->id(),
```

Comment:
could use html::getId();

Reply:
Addressed here: https://github.com/az-digital/az_quickstart/pull/1059/files#diff-4ebed9b1a9ecd773b91c632f8ef9212dc50473feba85ca4b9363a1f23c006bbfR70

And here: https://github.com/az-digital/az_quickstart/pull/1059/files#diff-3835e62d0ade8930c6536b446dfe773d6022f3eee035e46dbee2b12ebcbfb8afR177